src/wayback.ts

```diff
@@ -1,3 +1,4 @@
+import * as core from '@actions/core';
```

Comment:
I've got `allowSyntheticDefaultImports` set in `tsconfig.json`, so you should be able to do

```suggestion
import core from '@actions/core';
```